tests/app/data/test_schema_validation.py

```diff
@@ -83,6 +84,19 @@ def test_no_duplicate_ids_in_schema(self):
                     else:
                         unique_id.append(id_value)
 
+    def test_all_schemas_contain_confirmation_page(self):
+        schema_files = self.all_schema_files()
+
+        for schema_file in schema_files:
+            confirmation_or_summary_in_block = False
+            with open(schema_file, encoding="utf8") as file:
+                schema_json = load(file)
+                blocks = SchemaHelper.get_blocks(schema_json)
+                for block in blocks:
+                    if block['type'] in ["Summary", "Confirmation"]:
+                        confirmation_or_summary_in_block = True
```

Comment:
You can `break` after this, there's no need to keep looping once you've found one.